### DISCUSSION TITLE
What is the equivalent command for npm run?

### DISCUSSION THREAD

**Discussion by rfalanga** [coreybutler/nvm-windows#764] [Support] [answered]
I tried to find the answer to my question but couldn't see it here in the Discussions or Issues.

I am not an expert at npm, but I need to install npm to work on a tutorial. I searched and learned about nvm-windows, so decided to go with that.

The tutorial instructed me to run this command at the CLI prompt: `npm run dev`. So, I tried running `nvm run dev`, but that failed. Checking on the valid arguments for nvm-windows I see there is no `run` argument. I thought that perhaps `nvm use` might work, but that appears to me to specify which version of nvm-windows to use, not the same thing as specifying that I want to run the `dev` scripts in the Package.json file. So, what command do I use, please?

**Comment by rfalanga** [comment, marked as answer]
Never mind. I now see that npm was installed and I can just use the `npm run dev` command. Obviously, it was installed when I used nvm to install it.

I'm embarrassed.

As you were.

**Reply by coreybutler** [reply]
No worries, and don't be embarrassed. You're not the first and you won't be the last. Glad you got things working.
